### ISSUE TITLE
not working on OSX Sierra

### ISSUE THREAD

**Issue by petersgiles** [Makeblock-official/Makeblock-USB-Driver#5]
the drivers don't work on OSX Sierra

**Comment by fmarios** [comment]
I have the same issue. After upgrade to MacOS Sierra connecting mbot via usb causes kernel panic (computer freezes and then restarts). Do you have any solution? Thanks for any assistance.

**Comment by suhajdab** [comment]
I found [this](https://tzapu.com/ch340-ch341-serial-adapters-macos-sierra/), but not how to remove the old one.

**Comment by quocble** [comment]
Yes, do not follow the instructions for Sierra (which probably too late), you will brick your machine. You have to mount the drive, and remove usb.kext

**Comment by petersgiles** [comment]
> Yes, do not follow the instructions for Sierra (which probably too late), you will brick your machine. You have to mount the drive, and remove usb.kext

So can we have new instructions that give a step by step process for doing this

**Comment by quocble** [comment]
The driver mentioned by @suhajdab  worked for me. This one http://blog.sengotta.net/wp-content/uploads/2015/11/CH34x_Install_V1.3.zip

**Comment by adambisek** [comment]
This on works on OSX 10.13 High Sierra!
https://github.com/adrianmihalko/ch340g-ch34g-ch34x-mac-os-x-driver
